### PR TITLE
feat: add 'Blockchain Developer Path' section

### PR DESCRIPTION
**Pull request by mkubdev**
## Related Issue 

N/A

Closes: # 

### Describe the changes you've made

I added a new section for the website : 'Blockchain Developer Path' and cards with an hover effect.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
Capture | new |

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,24 +9,21 @@
...
-    <meta name="author" content="mkubdev" />
+    <meta name="author" content="@github:mkubdev" />
     <link rel="stylesheet" href="https://unpkg.com/tailwindcss/dist/tailwind.min.css" />
     <!--Replace with our tailwind.css once created, we use the base tailwind atm-->
     <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700" rel="stylesheet" />
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
-    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <link rel="icon" href="./assets/favicon.ico" type="image/x-icon">
```
Comment (Panquesito7) [inline index.html:19]: Wouldn't this be the correct code?
```suggestion
    <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
```

File: index.html [hunk 2]
```diff
@@ -87,20 +84,20 @@ <h1 class="my-4 text-5xl font-bold leading-tight">
           </p>
           <a href="https://github.com/web3community/start-here" target="_blank" rel="noopener">
             <button style="background: #0086D6" class="mx-auto lg:mx-0 hover:underline text-white rounded-md my-6 py-4 px-8 shadow-lg focus:outline-none focus:shadow-outline transform transition hover:scale-105 duration-300 ease-in-out">
-              Explore
+              Explore Now
```
Comment (Panquesito7) [inline index.html:87]: I think `Explore` sounds better. This doesn't have to be accepted, just a suggestion. 🙂
```suggestion
              Explore
```

File: index.html [hunk 3]
```diff
@@ -151,6 +148,100 @@ <h1 class="my-4 text-5xl font-bold leading-tight">
...
     </section>
 
+    <!--Blockchain Developer Path-->
+    <section class="bg-gray-50 py-8">
+      <div class="container max-w-7xl mx-auto m-8">
+        <h1 class="w-full my-2 text-4xl font-bold leading-tight text-center text-gray-800">
+          Blockchain Developer Path
+        </h1>
+        <p class="text-center text-gray-800 text-base px-6 mb-5 mx-10">
+          The latest field in tech industry
```
Comment (Panquesito7) [inline index.html:158]:
```suggestion
          The latest field in the tech industry
```

File: index.html (hunk 2 above)
Comment (mkubdev) [reply index.html]: I'm ok with your suggestion, think same. :)

File: index.html (hunk 1 above)
Comment (mkubdev) [reply index.html:19]: fixed!

**Review by mkubdev — COMMENTED** [review]
fix added

**Review by Panquesito7 — APPROVED** [review]
All is looking awesome! It'd also be great if we can see the deployment of the live website. 🙂
Thank you for your work! I'll let @vinzvinci review this PR before merging, though.

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Pull request merged**